spec/controllers/user_registrations_controller_spec.rb

```diff
@@ -2,6 +2,7 @@
 
 require 'spec_helper'
 
+# rubocop:disable Rails/ResponseParsedBody
 RSpec.describe UserRegistrationsController, type: :controller do
```

Comment:
> It looks like something that should be using `format.json` and that we should fix

Agreed, this should be fixed if possible.

Alternatively, we shouldn't disable the rubocop rule here. It's a violation that is still "to do"

Reply:
@rioug @dacook 

I will tackle it.

The UserRegistrationsController#create is not yet used for signup, I saw an [open issue](https://github.com/openfoodfoundation/openfoodnetwork/issues/4019) for `merging Spree::UserRegistrationsController into UserRegistrationsController`, so changing the format won't break anything